app/posts/apply-for-teacher-training/2020-06-18-international-candidates.md

```diff
@@ -0,0 +1,244 @@
+    caption: "* British and Irish – no further questions are asked\n* Other – candidate asked to select nationality from structured list\n* I have multiple nationalities – candidate can give details as free text"
+  }, {
+    text: "Do you have the right to work or study in the UK? (Yes)",
+    caption: "Candidates who are not British or Irish are asked to give details about their residency status, if they know it. They can give details of their residency, or indicate that this is something they will need to consider later."
+  }, {
+    text: "Do you have the right to work or study in the UK? (Not yet)",
+    caption: "They can also indicate that this is something they will need to consider later by selecting ‘Not yet’ or ‘I do not know’."
+  }, {
+    text: "Reviewing personal details"
+  }]
+}) }}
+
+## International addresses
+
+This section remains unchanged from the [original design for capturing international addresses](/apply-for-teacher-training/international-address/). We can derive a candidate’s domicile from the information given in this section.
+
+## English language proficiency
+
+Our [original design for English language proficiency](/apply-for-teacher-training/apply-launch/#30-personal-details) asked candidates to tell us if their main language is English. If yes, we asked them if they spoke any other languages. If no, we asked for details of any English language qualifications they have, as well as other languages spoken.
+
+This design caused a number of questions.
+
+What counts as a main language?
+: In our user research, international candidates found this question confusing. [See P1 on Lookback](https://lookback.io/watch/XWuzeuQ5Rky8QSBCg).
+
+Do we really need to ask about other languages spoken?
+: Or is this only relevant for certain courses/providers?
+
+Does it give providers the information they need?
+: The original design did not capture English language qualifications in a structured way. Providers tend to look for evidence of ability – such as a GCSE English equivalent or an English language qualification such as IELTS or TOEFL – so structured data is useful.
+
+### User needs
+
+{{ appUserNeeds({
+  items: collections['user-need'] | slugs(['AN026'])
+}) }}
+
+{{ appUserNeeds({
+  items: collections['user-need'] | slugs(['MN015'])
+}) }}
+
+### Hypothesis
+
+If we allow international candidates to enter an English language qualification
+Then they can better understand the eligibility criteria and what’s needed of them
+We’ll know this works when providers get the information they need to decide whether to make an offer.
+
+{{ appScreenshots({
+  headingLevel: 3,
+  items: [{
+    text: "Your application - qualifications",
+    caption: "If the nationality is not British or Irish, we show an ‘English language qualifications’ section under the qualifications title on the application."
+  }, {
+    text: "English language qualifications",
+    caption: "The 2 most common qualifications are shown."
```

Comment:
What do I do if I am not British or Irish, but I am from a country where I speak English? For example Canada.
Can I mark this section as complete without adding anything?

Reply:
That came up today. Does English GCSE qualification or equivalent also count as part of this assessment? You could also make the opposite assumption that someone who comes from an English speaking country has English as their first language, but that may not be true.

Reply:
(This section is mandatory, by the way… though it might be viewed as such)

Reply:
I think we should reconsider the language qualifications/ English GCSE section: see my idea on Trello ticket here: https://trello.com/c/pIrwuAyp/1687-design-%F0%9F%8C%90-review-international-features-with-international-team